tests/manage/mcg/test_bucket_policy.py

```diff
@@ -311,8 +311,421 @@ def test_anonymous_read_only(self, mcg_obj, bucket_factory):
+            user_list=obc_obj.obc_account,
+            actions_list=['GetObject'],
+            resources_list=[f'{obc_obj.bucket_name}/{object_key}'],
+            effect="Deny"
+        )
+        bucket_policy = json.dumps(bucket_policy_generated)
+
+        logger.info(f'Creating bucket policy on bucket: {obc_obj.bucket_name} with principal: {obc_obj.obc_account}')
+        assert helpers.put_bucket_policy(mcg_obj, obc_obj.bucket_name, bucket_policy), "Failed: PutBucketPolicy"
+
+        # Getting Policy
+        logger.info(f'Getting bucket policy from bucket: {obc_obj.bucket_name}')
+        get_policy = helpers.get_bucket_policy(mcg_obj, obc_obj.bucket_name)
+        logger.info('Got bucket policy:%s' % get_policy['Policy'])
+
+        # Verifying whether Get action is denied access
+        logger.info(f'Verifying whether user: {obc_obj.obc_account} is denied to GetObject')
+        try:
+            helpers.s3_get_object(obc_obj, obc_obj.bucket_name, object_key)
+        except boto3exception.ClientError as e:
+            logger.info(e.response)
+            response = HttpResponseParser(e.response)
+            if response.error['Code'] == 'AccessDenied':
+                logger.info('GetObject action has been denied access')
+            else:
+                raise UnexpectedBehaviour(f"Response received invalid error code {response.error['Code']}")
+
+        # Admin sets a new policy on same obc bucket with same account but with different action and resource
+        bucket_policy_generated = gen_bucket_policy(
+            user_list=obc_obj.obc_account,
+            actions_list=['DeleteObject'],
+            resources_list=[f'{obc_obj.bucket_name}/{"*"}'],
+            effect="Deny"
+        )
+        bucket_policy = json.dumps(bucket_policy_generated)
+
+        logger.info(f'Creating bucket policy on bucket: {obc_obj.bucket_name}')
+        assert helpers.put_bucket_policy(mcg_obj, obc_obj.bucket_name, bucket_policy), "Failed: PutBucketPolicy"
+
+        # Getting Policy
+        logger.info(f'Getting bucket policy from bucket: {obc_obj.bucket_name}')
+        get_policy = helpers.get_bucket_policy(mcg_obj, obc_obj.bucket_name)
+        logger.info('Got bucket policy:%s' % get_policy['Policy'])
+
+        # Verifying whether delete action is denied
+        logger.info(f'Verifying whether user: {obc_obj.obc_account} is denied to Get object')
+        try:
+            helpers.s3_delete_object(obc_obj, obc_obj.bucket_name, object_key)
+        except boto3exception.ClientError as e:
+            logger.info(e.response)
+            response = HttpResponseParser(e.response)
+            if response.error['Code'] == 'AccessDenied':
+                logger.info('Get Object action has been denied access')
+            else:
+                raise UnexpectedBehaviour(f"Response received invalid error code {response.error['Code']}")
+
+    @pytest.mark.polarion_id("OCS-2149")
+    @tier1
```

Comment:
let's have less tests in tier1. Please pick 1-3 test cases and change the rest to tier2 or tier3, according to tiers definition

Reply:
Moved tests to tier2, they are not that critical.